src/onegov/org/cronjobs.py

```diff
@@ -1011,3 +1014,98 @@ def send_push_notifications_for_news(request: OrgRequest) -> None:
         print(f'Error sending notifications: {e}')
+
+
+@OrgApp.cronjob(hour=3, minute=0, timezone='Europe/Zurich')
+def normalize_adjacency_list_order(request: OrgRequest) -> None:
+    """Normalizes the 'order' column for all AdjacencyList subclasses.
+
+    The midpoint insertion strategy for 'order' (Decimal) can lead to
+    precision issues or very close values over time. This cronjob
+    renumbers the 'order' for each group of siblings (same parent_id)
+    sequentially starting from 1, effectively resetting the order values
+    while preserving the relative order within each sibling group.
+    """
+    session = request.session
+    processed_tables = set()
+
+    def is_concrete_subclass(cls: type) -> bool:
+        """Check if a class is a non-abstract subclass of AdjacencyList."""
+        return (
+            issubclass(cls, AdjacencyList)
+            and cls is not AdjacencyList
+            and not isabstract(cls)
+        )
+
+    # Find all relevant model classes
+    adjacency_subclasses = set(
+        chain.from_iterable(
+            find_models(base, is_concrete_subclass)
+            for base in request.app.session_manager.bases
+        )
+    )
+
+    log.info(f'Found {len(adjacency_subclasses)} '
+        'potential AdjacencyList models.')
+
+    for model in adjacency_subclasses:
+        mapper = inspect(model)
+        table = mapper.local_table  # Get the mapped table
+
+        # Basic sanity checks for the model's mapping
+        if (table is None or not mapper.primary_key or
+            len(mapper.primary_key) != 1):
+            continue
+
+        table_name = table.name
+        pk_column_name = mapper.primary_key[0].name
+
+        # Check if already processed or missing required columns
+        if table_name in processed_tables:
+            continue
+        if 'order' not in table.columns or 'parent_id' not in table.columns:
+            log.warning(
+                f"Skipping table '{table_name}': Missing 'order' or "
+                f"'parent_id' column."
+            )
+            continue
+
+        # Use ROW_NUMBER() partitioned by parent_id to generate new order
+        # values starting from 1 for siblings under the same parent.
+        # Rows with NULL parent_id (root nodes) are ignored
+        update_sql = text(
+            f"""
+            with numbered_siblings as (
+                select
+                    "{pk_column_name}",
+                    row_number() over (
+                        partition by "parent_id"
+                        order by "order" asc nulls last
+                    ) as new_order
+                from
+                    "{table_name}"
+                where
+                    "parent_id" is not null
+            )
+            update "{table_name}"
+            set "order" = numbered_siblings.new_order
+            from numbered_siblings
+            where "{table_name}"."{pk_column_name}" =
+                numbered_siblings."{pk_column_name}";
```

Comment:
```suggestion
            WITH numbered_siblings AS (
                SELECT
                    "id",
                    ROW_NUMBER() OVER (
                        PARTITION BY "parent_id"
                        ORDER BY "order" ASC NULLS LAST
                    ) AS new_order
                FROM
                    "{table_name}"
                WHERE
                    "parent_id" IS NOT NULL
            )
            UPDATE "{table_name}"
            SET "order" = numbered_siblings.new_order
            FROM numbered_siblings
            WHERE "{table_name}"."id" =
                numbered_siblings."id";
```

Technically this can break if `table_name` contains a quote. But it should be fine. We can switch to `sqlalchemy.update` on the table/column object if it becomes an issue. There's probably also an escape function we could use to escape the table name (there definitely is in psycopg, but we might not want to use their API, since we may switch backends, so we should be backend agnostic where possible).